README.md

```diff
@@ -20,6 +20,14 @@ Dates are subject to change.
 
 The Release schedule is available also as a [JSON][] file.
 
+### Release Phases
+
+There are three phases that a Node.js release can be in: 'Current', 'Active Long-term Support', and 'Maintenance'. Odd-numbered release lines are not promoted to LTS - they will not go through the 'Active LTS' or 'Maintenance' phases. 
```

Comment:
Nit: it would be good to keep the 80 character limit as in the rest of the file.